py/tests/unit/with_runtime_sparkling/test_Spark_to_H2O_conversions.py

```diff
@@ -277,7 +277,7 @@ def testConvertTimeValueFromSparkToH2OAndBack(spark, hc, timeZone, sparkType):
 
     hf = hc.asH2OFrame(df)
     hfResultString = hf.__unicode__()
-    hfParsedItems = hfResultString.split('\n')[2:6]
+    hfParsedItems = hfResultString.split('\n')[1:5]
```

Comment:
I'm not sure the code is expressive enough - the specified upper bound seems weird to me. I would probably wrote something like:
`# skip column name`
`hfParsedItems = hfResultString.split('\n')[1:]`

Reply:
We need to skip also the footer `[4 rows x 1 column]`. 

> I'm not sure the code is expressive enough

Imho, we don't need to be worried about performance if dataset has just six rows.

Reply:
Well, I missed the footer but I still think the sequence
`# skip header and footer`
`hfParsedItems = hfResultString.split('\n')[1:-1]`
is easier to understand for an occasional reader than
`hfParsedItems = hfResultString.split('\n')[1:5]`

Reply:
Changed to `hfResultString.split('\n')[1:-2]`